addons/udes_security/models/ir_attachment.py

```diff
@@ -127,6 +136,19 @@ def _remove_exif_data(self, datas):
         image.save(output, format=image.format)
         return base64.b64encode(output.getvalue())
 
+    def _process_url_image(self, url):
+        """Fetch an image from the URL and convert it to binary"""
```

Comment:
At this point, is URL guaranteed to be safe (so a UDES URL I suppose)?  Or does it matter?

Reply:
The tests appear to be happy if we add a check like this:
```python
if not vals["url"].startswith("/"):
```
to avoid checking relative urls.

Still need to look into the checking of type though, as I'm not sure if the _remove_exif_data function will get hit, is "mimetype" going to be set to one of the allowed mimetypes if it's a URL?

Reply:
I spun up a unit test locally and it seems to go through the mimetype checks.

Reply:
I'm happy with the latest change